Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyProductEvents.Codeunit.al

```diff
@@ -347,4 +347,21 @@ codeunit 30177 "Shpfy Product Events"
     internal procedure OnBeforeSendUpdateShopifyProductVariant(ShopifyShop: Record "Shpfy Shop"; var ShopifyVariant: Record "Shpfy Variant"; xShopifyVariant: Record "Shpfy Variant")
     begin
     end;
-}
\ No newline at end of file
+
+    [IntegrationEvent(false, false)]
+    /// <summary> 
+    /// Raised Before Modify on Items.
+    /// </summary>
+    internal procedure OnDoUpdateItemVariantBeforeModify(Shop: Record "Shpfy Shop"; var ShopifyProduct: Record "Shpfy Product"; ShopifyVariant: var Item: Record Item; var IsModified: Boolean);
```

Comment:
I think we should stick to similiar names like the other events, that use `ShopifyShop` for `Record "Shpfy Shop"`.

Additionaly this event looks a little bit wrong. The `ShopifyVariant:` should be removed from the signature

Reply:
I agree fully on your suggestions! I have changed as according and committed the changes.